setup.py

```diff
@@ -41,6 +41,9 @@
 # Global variables for controlling the build variant
 ################################################################################
 
+# Default value is set to TRUE\1 to keep the settings same as the current ones.
+# However going forward the recomemded way to is to set this to False\0
+USE_MSVC_STATIC_RUNTIME = bool(os.getenv('USE_MSVC_STATIC_RUNTIME', '1') == '1')
```

Comment:
>USE_MSVC_STATIC_RUNTIME  [](start = 0, length = 24)

This is would be false by default here?

Reply:
USE_MSVC_STATIC_RUTIME will be tru by default which means we will use /MT option (current option) by default

---
In reply to: [346494736](https://github.com/onnx/onnx/pull/2452#discussion_r346494736) [](ancestors = 346494736)